src/app/templates/app/archive_requests.html

```diff
@@ -7,6 +7,17 @@
 
 <div id="messages" class="messages">
 </div>
+{% if github_login  %}
+<p id="githubLogin" hidden>True</p>
+{% else %}
+<p id ="githubLogin" hidden> False </p>
+{% endif %}
+{% if authorized  %}
+<p id="authorized" hidden>True</p>
+{% else %}
+<p id ="authorized" hidden> False </p>
+{% endif %}
+
```

Comment:
Totally inefficient and insecure way of doing this.
Think about it this way, assume I have access to "Inspect Element" function in my browser. If I change this html tag value from False to True, will I be able to access the archive request feature?